jm-glass/src/ChordPage.tsx

```diff
@@ -0,0 +1,99 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { CHORDS, StringedInstrument } from 'jmusic-engine';
+import { Chord, ChordSelector, NoteSelector } from './components';
+import {
+  ChordContainer,
+  Header,
+  HeaderContainer,
+  Logo,
+  Page,
+  PagesBar,
+  PagesContainer,
+} from '@design/Theme';
+
+const CACHE_HOST = '/jmusic/cache';
+const guitar = StringedInstrument.guitar;
+
+interface SimplifiedChord {
```

Comment:
`SimplifiedChord` type can be used for the `Chord` component too.
`Chord` component is defining the type again.
Maybe we can put `SimplifiedChord` in a separate file and export it to reuse.